src/QuickCheck/Console/TestCommand.php

```diff
@@ -58,7 +60,7 @@ private function executeTests(Property $property, OutputInterface $output) {
         $startTime = microtime(true);
         $rng = new Random($this->seed);
         /** @var ShrinkTreeNode[] $tests */
-        $tests = Lazy::take($this->numTests, $property->randomTests($rng, $this->maxSize));
+        $tests = Lazy::take($this->numTests, $property->randomTests($rng));
```

Comment:
This is a bit weird, `maxSize` is no longer used after this. Perhaps we should use `$property->withMaxSize($this->maxSize)` here:

https://github.com/steos/php-quickcheck/blob/ff14a5946f7033315bb05dc84c270fe27025930f/src/QuickCheck/Console/TestCommand.php#L153

Reply:
Yeah, after just looking at this for a few minutes I think this whole code is not used at all anymore. If I remember correctly, this was the original attempt to implement the test runner as symfony console command but I had some issues getting it to work exactly how I wanted and then abandoned that approach (not sure anymore what the issue was though). 
It looks to me the actual implementation that's used is in src/QuickCheck/TestRunner.php and this console command code can probably be deleted.